tools/devtool

```diff
@@ -581,6 +581,38 @@ apply_linux_61_tweaks() {
 }
 
 
+apply_performance_tweaks() {
```

Comment:
Can you add messages for the configurations we are touch here, so that we can verify that they have been applied during a run?

Reply:
Done!